src/Author.java

```diff
@@ -0,0 +1,17 @@
+public class Author {
+    String name;
+    String surname;
+
+    public String getName() {
```

Comment:
private String firstName;
    private String lastName;

    public Author(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }